Dokument/Dateien/Magie_Essenz.tex

```diff
@@ -1,103 +1,876 @@
+	\item ...
 \end{itemize}
 
-\subsubsection{Elektrizität und Magnetismus}\label{sec:elektromagie}
+\subsubsection{Meisterliche Beherrschung} 
 \begin{itemize}
-	\item Das Beeinflussen elektrischer und magnetischer Ströme/Ladungen
-	\item Intuitiv das Abgeben eines elektrischen Schocks oder kleine Bewegungen von magnetischem Material
-	\item Ausbaubar bis in zur Beeinflussung von elektrischen Strömen an Nervenzellen (Signalweiterleitung mittels Neuronen), (ent-)magentisierung von Metallen, Kontrolle über die Bewegung von Metallen, Blitze ableiten, WIEDERBELEBUNG!? (unter entsprechenden Umständen)
-	\item Wiederbelebung: so weit kam nur ein Zauberer, welcher schon verstorben ist: konnte manchmal Leute zurück ins Leben bringen, wenn es noch dicht genug am Todeszeitpunkt dran war -> war sozusagen ein Defibrillator
+	\item Das Blut/der Körper des Gegners kann zum Kochen gebracht werden.
+	\item Körperteile des Gegners können erfroren werden.
+	\item ...
 \end{itemize}
+Zurück zu Tab. \ref{tab:magie-verbreitung}.
 
-\subsubsection{Absorption oder Nullmagie}\label{sec:nullmagie}
-\begin{itemize}
-	\item eine besondere Art von Magie ist die "Nullmagie" (andere Namen möglich). Diese Magieart ist sozusagen Anti-Magie, denn der Träger ist nicht fähig, Magie zu wirken, sondern nur zu absorbieren und damit ihre Effekte zu verhindern. Ebenso kann keine Magie auf ihn gewirkt werden. Er neutralisiert und absorbiert Magie - kann diese absorbierte Magie aber auch nicht nutzen, sondern die verschwindet gleich wieder aus seinem Körper
-	\item zum aktuellen Zeitpunkt nicht existent. Entsteht erst bei der Gottwerdung in den Überlebenden. Von da an rezessiv vererbbar
-\end{itemize}
 
 
-\chapter{Essenz}
-\subsection{Was ist Essenz?}
-\begin{itemize}
-	\item Lebewesen besitzen eine Essenz, eine Art Seele (wird im Folgenden als Seele bezeichnet, um die Erklärung einfacher zu halten). Diese Seele umfasst das, was dieses Lebewesen "sein eigen" nennt im Sinne des Körpers, der Existenz. Das ist im Normalfall der Körper. Könnte aber auch eine Prothese am Körper umfassen, wenn das Lebewesen diese als Teil seines Körpers begreift.
-\end{itemize}
\ No newline at end of file
+\subsection{Terahertz}\label{magie:terahertz}
+Mittels Terahertz-Strahlung lässt sich wenige Zentimeter in weiches bzw. organisches Gewebe blicken, ohne Strahlungsschäden zu riskieren.
+Diese Wellenlänge wird allerdings schon von Wasser absorbiert.
+Das erklärt die geringe Reichweite und Unschädlichkeit.
+
+\subsubsection{Verbreitung}
+Gerade unter den intelligenteren Tieren verbreitet, aufgrund des guten intuitiven Nutzens.
+
+\subsubsection{Bedeutung im Spiel}
+Je nachdem inwiefern es den Menschen in Mantodea zur Verfügung steht. %todo
+
+\subsubsection{Intuitive Nutzung}
+Diese Fähigkeit ermöglicht es, sehr schnell Parasiten zu finden oder sich der genaueren Natur einer Verletzung gewahr zu werden.
+
+\subsubsection{Erlernbare Steigerungen}
+\begin{outline}
+	\1 ...
```

Comment:
+ evtl Material-/Probenanalyse zum einen in Handwerksberufen, zum anderen in der Alchemie und Medizin (z.B. Salben, Kräuter, Gifte, etc.)